compiler/testData/diagnostics/tests/platformTypes/nullabilityWarnings/safeCall.fir.kt

```diff
@@ -24,20 +24,20 @@ fun test() {
     // platform type with no annotation
     val platformJ = J.staticJ
 
-    platformNN?.foo()
+    platformNN<!UNNECESSARY_SAFE_CALL!>?.<!>foo()
     platformN?.foo()
-    platformJ?.foo()
+    platformJ<!UNNECESSARY_SAFE_CALL!>?.<!>foo()
```

Comment:
This is exactly same as https://github.com/JetBrains/kotlin/pull/4257/files#r604619673, and after digging a bit more, I think I found a root cause, but it requires a separate PR/commit I think. Will update here and that PR as well.

Reply:
Discussed at https://github.com/JetBrains/kotlin/pull/4257#discussion_r605431455; will be fixed like https://github.com/JetBrains/kotlin/pull/4269; and on top of that, this one is gone, and now it's `// FIR_IDENTICAL`.